### DISCUSSION TITLE
Welcome to awesome-software-architecture Discussions!

### DISCUSSION THREAD

**Discussion by mehdihadeli** [mehdihadeli/awesome-software-architecture#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Hadi-ctr** [comment]
Am Hadi ctr

**Comment by anderson828** [comment]
Hello

**Comment by mehdihadeli** [comment]
Hi everyone :)

**Comment by z1lV3r** [comment]
Hey, I just discover this repo and I really appreciate that you're sharing this. Thanks a lot!

**Reply by mehdihadeli** [reply]
You're welcome :)
